### PR TITLE
Add troubleshooting, update tested ver, adjusted tested area

### PR DESCRIPTION
**Pull request by asheroto**
- Add troubleshooting for Windows 10 and made it its own section
- Updated tested version to latest (confirmed)
- Adjusted tested area text

If you could add information on how to configure a debugger on Visual Studio Code I see many devs benefiting from that as its a popular IDE now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +12,15 @@ This environment includes the latest versions of the following software:
...
+ - Docker - https://www.docker.com/products/docker-desktop
+ - Tested and confirmed working on:
+	 - macOS 10.15 (Catalina)
+	 - Windows 10 Version 21H1 build 19043.1165
 
 ## Running the Container
 
 1. Using composer: `composer create-project jlucki/docker-php-dev-env <project name>` or download the files from this repo and copy them to your project directory
-3. Run `docker-compose up` from the project directory
+3. Run `docker compose up` from the project directory
```
Comment (jlucki) [inline README.md:23]: I'd add a note here to use `docker-compose up` on versions older than 3.6, `docker compose` (no dash), is only availble on 3.6 and up https://docs.docker.com/compose/cli-command/

**Review by jlucki — CHANGES_REQUESTED** [review]
Nice, thanks for taking the time to do this.

**Comment by jlucki** [comment]
I've not used VS Code so I'm not familiar with the setup process on there. I might give it a try one day though.

**Comment by asheroto** [comment]
> I've not used VS Code so I'm not familiar with the setup process on there. I might give it a try one day though.

It is marvelous.  I was using a few other IDE's for a long time.  One day tried VS Code and never looked back.  What makes it so powerful in my opinion is the vast amount of [extensions](https://marketplace.visualstudio.com/VSCode).

**Review by jlucki — APPROVED** [review]

**Pull request merged**